tests/cc/test_brb.py

```diff
@@ -109,7 +109,7 @@ def connect_ports(self, prog_id_pem, prog_id_br, curr_pem_pid, curr_br_pid,
         br_dest_map[c_uint(curr_br_pid)] = br_dest_map.Leaf(prog_id_pem, curr_pem_pid)
         self.pem_port[c_uint(curr_pem_pid)] = c_uint(ifindex)
         self.pem_ifindex[c_uint(ifindex)] = c_uint(curr_pem_pid)
-        mac_addr = br_mac_map.Key(int(EUI(vm_mac.decode())))
+        mac_addr = br_mac_map.Key(int(EUI(vm_mac)))
```

Comment:
You are correct. It is my bad. Just tried and works perfectly fine.

On Mon, Jul 13, 2015 at 9:59 PM, Brenden Blanco <notifications@github.com>
wrote:

> In tests/cc/test_brb.py
> <https://github.com/iovisor/bcc/pull/85#discussion_r34537325>:
>
> > @@ -109,7 +109,7 @@ def connect_ports(self, prog_id_pem, prog_id_br, curr_pem_pid, curr_br_pid,
> >          br_dest_map[c_uint(curr_br_pid)] = br_dest_map.Leaf(prog_id_pem, curr_pem_pid)
> >          self.pem_port[c_uint(curr_pem_pid)] = c_uint(ifindex)
> >          self.pem_ifindex[c_uint(ifindex)] = c_uint(curr_pem_pid)
> > -        mac_addr = br_mac_map.Key(int(EUI(vm_mac.decode())))
> > +        mac_addr = br_mac_map.Key(int(EUI(vm_mac)))
>
> No, it looks like you have a typo. Did you type or apply the full patch?
> It should not be "vm_mac()" with the parentheses.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iovisor/bcc/pull/85/files#r34537325>.
>